webui/react/src/utils/browser.ts

```diff
@@ -106,7 +106,7 @@ const updateFavicon = (iconPath: string): void => {
   linkEl.href = iconPath;
 };
 
-export const updateFaviconType = (active: boolean, branding = BrandingType.Determined): void => {
+export const updateFaviconType = (active: boolean, branding: BrandingType): void => {
```

Comment:
You removed a default argument here, was that intentional?

Reply:
yes, it is used in only a place and value is set